yt/docs/README.md

```diff
@@ -211,3 +437,47 @@ yt check-permission yql write //tmp
     "subject_name" = "yandex";
 }
 ```
+````
+
+------
+
+### Documentation Philosophy
+
+- When writing text, determine your target audience, whom you are writing for. Take into account that YTsaurus users may have different backgrounds and may be from different areas: analysts, engineers, administrators, and developers. The target audience influences the level of detail of your text and the terminology you use.
+- Your text should not disrupt the integrity of the entire document or its narrative logic. When writing content, consider the context of the section where this content is intended to be added.
+- Try to avoid jargon, slang, Anglicisms, and colloquial language. Use commonly accepted terms. Strive to follow the academic style.
+- Accompany your text with examples. Content without examples is perceived as abstract and is hard to assimilate.
+- Use cross-links between the sections.
+- Aim to write concisely and clearly. Do not make sentences too long — up to 15 words per sentence. Split long paragraphs into short ones — up to 5 sentences in one paragraph.
+- If you need any help with content writing, don't hesitate to ask questions in YTsaurus [community chats](#need-help).
```

Comment:
1. Seems like an important paragraph that we may consider moving upwards
2. It is interesting that we break rules that we define in the same document :)
   + Maybe we should change/remove the rule?
   + Same for unordered lists above
> items in an unordered list start with a lowercase letter and end with a semicolon;